### PR TITLE
Update penalties.mdx

### PR DESCRIPTION
**Pull request by gtelnet**
dkg was changed to `5.0` as of block 1,107,995

### CODE REVIEW

**Comment by gtelnet** [comment]
@PharkMillups are we supposed to tag you here to get them reviewed?

**Comment by PharkMillups** [comment]
@gtelnet Whoops. Normally I pay attention but this one slipped through. Sorry. Reviewing now.

File: docs/mine-hnt/validators/penalties.mdx [hunk 1]
```diff
@@ -68,9 +68,10 @@ To ensure that consensus groups members are continually churned, and that no ind
 A DKG penalty can occur if an election fails, and the Validator was part of the reason for the election to fail. This can occur throughout the dkg process or signature phase.
 
 - At the end of the election, all Validators vote and sign the election result. If the election fails, Validators that a majority agree did not sign the election results are penalized.
+- Failed elections are significant to the network as they prevent new groups from being elected and rewards to be issued to all network participants including Hotspots.
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/penalties.mdx]:
```suggestion
- Failed elections are significant to the network as they prevent new groups from being elected and rewards from being issued to all network participants including Hotspots.
```

**Comment by PharkMillups** [comment]
One small suggested edit. Get that merged and we'll deploy. Also ping me on discord so we can get you the contributor tag.

**Review by PharkMillups — APPROVED** [review]

**Pull request merged**